modules/system/classes/UpdateManager.php

```diff
@@ -532,11 +530,14 @@ public function updatePlugin($name)
     }
 
     /**
-     * Removes an existing plugin
+     * Rollback an existing plugin
+     *
      * @param string $name Plugin name.
+     * @param string $stopOnVersion If this parameter is specified, the process stops after
+     * the specified version is rolled back.
      * @return self
      */
-    public function rollbackPlugin($name)
+    public function rollbackPlugin(string $name, string $stopOnVersion = null)
```

Comment:
I'm not sure I fully understand this, if I run `php artisan plugin:rollback Acme.Demo 1.5.7` is `Acme.Demo` going to be on `1.5.7` or `1.5.6` / whatever version immediately preceded `1.5.7`?

Reply:
1.5.7

Reply:
@prhost in that case the docblock comment is wrong

Reply:
fixed